test/Libraries/CommandLineTests/CommandLineTests.cs

```diff
@@ -0,0 +1,171 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+using NUnit.Framework;
+using DynamoSandbox;
+using System.Text.RegularExpressions;
+using System.Xml;
+using Dynamo;
+using Dynamo.Applications.StartupUtils;
+using DynamoCLI;
+
+namespace Dynamo.Tests
+{
+   
+    internal class CommandLineTests : DynamoModelTestBase
+    {
+
+        protected override void GetLibrariesToPreload(List<string> libraries)
+        {
+            libraries.Add("VMDataBridge.dll");  // Required for Watch node.
```

Comment:
Do you need to load all of these libraries for your tests? Load only the bare minimum required to keep testing time to a minimum.